static/docs/user-guide/dvcignore.md

```diff
@@ -13,17 +13,17 @@ files handling. `.dvcignore` by design works similar way as `.gitignore` does.
 
 - You need to create `.dvcignore` file.
 - Populate it with [patterns](https://git-scm.com/docs/gitignore) that you would
-  like to ignore
+  like to ignore.
 - Each line should contain only one pattern.
 - During execution of commands that traverse directories, DVC will ignore
   matching paths.
-- Ignored files will not be cached, they will be non-existent for DVC. It's
-  worth to remember that, especially when ignoring files inside DVC-handled
-  directories.
-- Keep in mind, that when you add to dvcignore entries that affect one of the
+- Ignored files will not be saved in cache, they will be non-existent for DVC.
```

Comment:
I would also split basic "create file, add item, profit" part from the more descriptive items - like how it works. It can be two different sections.